src/entities/api.entity.ts

```diff
@@ -21,7 +21,7 @@ export class Api extends SharedEntity {
   description: string;
 
   @Column()
-  base_url: string;
+  base_url?: string;
```

Comment:
You modified entity? no nah

Reply:
my bad, changed

Reply:
@danifara3 these fields u made optionally are actually required